packages/v4/src/content/design-guidelines/components/menu/menu.md

```diff
@@ -0,0 +1,82 @@
+---
+id: Menu
+section: components
+---
+
+A menu component that can be used in a variety of contexts whenever the user needs to choose between multiple values, options, or actions. A menu can be opened in a [dropdown](/components/dropdown), a [select list](/components/select), or it can be revealed upon right clicking on a region within a page.
+
+## Elements and variations
+### Action menu
+An actions menu is used for presenting a list of actions or links.
+
+<img src="./img/basic-menu.png" alt="actions dropdown" width="863"/>
+
+1. **Menu item:** Menu items represent discrete actions that can be taken. Selecting a menu item inside a dropdown will trigger the action and dismiss the menu, while selecting a menu item inside a select list will select the item.
+
+2. **Icon (optional):** You may place an icon before each menu item to aid in recognition. It has been shown that familiar icons speed the recognition of text labels, but only use icons if they will be easily recognized and are distinguishable. Never use icons simply as a decoration.
+
+3. **Links:** Links may be used in place of or mixed with actions to navigate the user to a new page. If the link will open in a new window, the external link icon should be used to annotate the link and let the user know to expect this behavior.
+
+4. **Descriptions (optional):** Descriptive text may be added below each menu item. This should only be used when the menu item label by itself may not be clear to all users. Keep descriptive text to two lines or less.
+
+### Option select menus
+Use an option select menu when you want to persist the selected items. This is typically the case for select menus or settings menus that allow users to select between multiple options. See the [Select](/components/select) and [Options menu](/components/options-menu) components for more details. You may present a single or multiple groups of options within the same menu.
+
+<img src="./img/option-select-menu.png" alt="option select menus" width="863"/>
```

Comment:
Are the checkboxes in these images meant to be grey? Or should they be blue? I know they are grey in the symbol library but I feel that this may be a bug. Been meaning to bring this up to Marie!

Reply:
Yeah, in the component examples they are blue. I can fix these, but it should also be fixed in the Sketch library.